data-raw/ni-beds.R

```diff
@@ -0,0 +1,44 @@
+# Northern Ireland on Health Inequalities Explorer
+# Secondary care indicators - bed availability
+
+# ---- Load libs ----
+library(tidyverse)
+library(devtools)
+library(lubridate)
+
+# ---- Load internal sysdata.rda file with URL's ----
+load_all(".")
+
+query_url <-
+  query_urls |>
+  filter(id == "ni_beds") |>
+  pull(query)
+
+# ---- Download and wrangle data ----
+raw <- read_csv(query_url, skip=2)
+
+# Change dataset to tidy data with all relevant information
+# Removed "Day Case" column since it included missing values and duplicate values with "Total Day Cases".
+# Can calculate number of discharge beds = total available beds - total occupied beds
+ni_beds <- raw |>
+  select(
+    financial_year = "Financial Year",
+    quarter_ending = "Quarter Ending",
+    HSC = "HSC Trust",
```

Comment:
Please match this column name to the same key used throughout the other humaniverse packages, e.g., use `trust18_code` instead.